scripts/upload_image.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+"""
+Upload an image to Google Cloud.
+
+To use this:
+- Install the Google Cloud CLI (https://cloud.google.com/storage/docs/gsutil_install)
+- Set up credentials (https://cloud.google.com/storage/docs/gsutil_install#authenticate)
+"""
+
+import argparse
+import hashlib
+import os
+import subprocess
+
+
+def content_hash(path: str) -> str:
+    h = hashlib.sha1()
+    b = bytearray(128 * 1024)
+    mv = memoryview(b)
+    with open(path, "rb", buffering=0) as f:
+        while n := f.readinto(mv):
+            h.update(mv[:n])
+    return h.hexdigest()
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Upload an image.")
+    parser.add_argument("file", type=str, help="Path to the image.")
+    args = parser.parse_args()
+
+    object_name = content_hash(args.file) + f"_{os.path.basename(args.file)}"
+
+    subprocess.check_output(["gsutil", "cp", args.file, f"gs://rerun-static-img/{object_name}"])
```

Comment:
If you didn't want to use sub-process we could consider doing this directly with the google storage APIs, though it's a bit more verbose...
```
from google.cloud import storage
gcs_client = storage.Client()
static_bucket = gcs_client.bucket("rerun-builds")
destination = static_bucket.blob(object_name)
destination.upload_from_file(buffer)
```